mods/ts/sequences/structures.yaml

```diff
@@ -846,7 +846,7 @@ nahpad:
 		Offset: 0, 0
 		UseTilesetCode: false
 
-proc: # TODO: unused narefn_a, narefn_b
+proc:
```

Comment:
We want to get rid of those as they have to be manually reverted on rule upgrades.